scripts/update-drafts.ts

```diff
@@ -56,7 +74,31 @@ async function main() {
 
   // Build a map from URLs to spec.
   const pageToSpec = new Map<string, WebSpecsSpec>();
-  for (const spec of webSpecs) {
+
+  let selectedSpecs = webSpecs;
+  let selectedKeys = filterKeys;
```

Comment:
OK this almost works now! If you don't supply keys though, this value being undefined fails.

```suggestion
  let selectedKeys = filterKeys ?? [];
```

Reply:
Good catch! Thanks!